llvm/lib/Target/SPIRV/SPIRVSymbolicOperands.td

```diff
@@ -384,6 +384,7 @@ defm SPV_INTEL_2d_block_io : ExtensionOperand<122, [EnvOpenCL]>;
 defm SPV_INTEL_int4 : ExtensionOperand<123, [EnvOpenCL]>;
 defm SPV_KHR_float_controls2 : ExtensionOperand<124, [EnvVulkan, EnvOpenCL]>;
 defm SPV_INTEL_tensor_float32_conversion : ExtensionOperand<125, [EnvOpenCL]>;
+defm SPV_INTEL_arbitrary_precision_floating_point: ExtensionOperand<127, [EnvOpenCL]>;
```

Comment:
Future of Intel FPGA extensions was a bit uncertain. Now it's more clear - they are renamed to Altera extensions in https://github.com/KhronosGroup/SPIRV-Registry/commit/e86c766cfbff190912b48c6bddfec6292835b81a . I suggest to start generating new names for the extensions, capabilities and instructions.